pkg/volume/glusterfs/glusterfs_uid.go

```diff
@@ -0,0 +1,151 @@
+
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+
+/*
+
+Reference#
+https://github.com/openshift/origin/blob/master/pkg/security/uid/uid.go
+
+
+*/
+package glusterfs
```

Comment:
* This should be probably made public in `pkg/util/*`, where OpenShift could find it (so we don't need to maintain two copies in two projects).

* I'd appreciate if this file was "generic integer range/block" and we did not use terms like UID, GID or port here (also in the file name).

* I miss documentation and unit tests.

This can be probably implemented in a separate patch.